documentation/general/dash-sonic-hld.md

```diff
@@ -220,6 +243,8 @@ DASH_ENI_TABLE:{{eni}}
     "vnet": {{vnet_name}}
     "pl_sip_encoding": {{string}} (OPTIONAL)
     "pl_underlay_sip": {{ip_addr}} (OPTIONAL)
+    "v4_meter_policy_id": {{string}} (OPTIONAL)
+    "v6_meter_policy_id": {{string}} (OPTIONAL)
```

Comment:
why these are string instead of integer?

Reply:
these are meter policy UUIDs, similar to eni-id